### DISCUSSION TITLE
How do I change the name of SbieDll.dll ?

### DISCUSSION THREAD

**Discussion by krudge137** [sandboxie-plus/Sandboxie#2877] [Documents and notes]
Can someone help me with this please? is there a way to rename to something else without getting errors? Please help !

**Comment by isaak654** [comment]
https://github.com/sandboxie-plus/Sandboxie/issues/2310#issuecomment-1324543963

**Reply by krudge137** [reply]
I have tried this, but doesn't solve my issue, all I want is a rename I don't need to hide it

**Reply by isaak654** [reply]
https://github.com/sandboxie-plus/Sandboxie/discussions/2313#discussioncomment-3822620

**Reply by krudge137** [reply]
I'm a vs newbie 😞

**Reply by isaak654** [reply]
I hope this helps: https://github.com/sandboxie-plus/Sandboxie/discussions/1089#discussioncomment-2441183

**Reply by krudge137** [reply]
I appreciate your help man, I just have one question, do I have to download vs 2019? because I have already downloaded vs 2022?

**Reply by isaak654** [reply]
You might want to check issue #2562

**Reply by krudge137** [reply]
I compiled **Sandboxie** using the `Sandbox.sln`, and when I tried to run `SbieCtrl.exe` it showed me this error : 
`SBIE9234 Service startup error level 9361 status=C0000022 error=-1073741790` ( [Screenshot](https://i.imgur.com/w99zDMa.png) ) 
but I only compiled `Sandbox.sln` because I haven't installed Qt yet since it requires a decent amount of diskspace that I don't have at the moment. 
Does compiling the service requires Qt? 
I have compiled `Sandbox.sln` both with x32 & x64 and I used  the `merge_builds.cmd` located in the `Installer` folder
Here is a [Screenshot](https://i.imgur.com/NoZGfSC.png) of the files that were merged.
I'm pretty sure I'm doing something wrong but I don't seem to know which exactly since I'm new to VS. Thanks in advance !

**Reply by isaak654** [reply]
Did you enable test signing mode and rebooted? https://github.com/sandboxie-plus/Sandboxie/issues/1248#issuecomment-933379389

**Reply by krudge137** [reply]
I just tried it and installed the `.cer` file but I got some errors almost the same as before: 

[Screenshot 1](https://i.imgur.com/vFcFvio.png) [Screenshot 2](https://imgur.com/ngTKCgD) after I press `Ok` it shows this one:
```

SBIE9234 Service startup error level 9361 status=C0000022 error=-1073741790

SBIE9234 Service startup error level 9361 status=C0000022 error=-1073741790

SBIE1101 Sandboxie driver (SbieDrv) version 5.64.2 initialized

SBIE9234 Service startup error level 9361 status=C0000022 error=-1073741790

SBIE9234 Service startup error level 9361 status=C0000022 error=-1073741790

SBIE2331 Service start failed:  [22 / 1060] The specified service does not exist as an installed service


```

**Reply by isaak654** [reply]
> I haven't installed Qt yet since it requires a decent amount of diskspace that I don't have at the moment.
> I used the merge_builds.cmd located in the Installer folder

Installer folder contains only the files related to Sandboxie Plus, but they're not useful if you are not going to run the the Qt online installer and select all required components. The only alternative would be to follow the [Sandboxie Classic installer instructions](https://github.com/sandboxie-plus/Sandboxie/tree/master/Sandboxie/install#sandboxie-classic-installer-instructions).

Support request moved to https://github.com/sandboxie-plus/Sandboxie/issues/2880
